### ISSUE TITLE
Kobra Plus Bricked

### ISSUE THREAD

**Issue by Rage496** [wabbitguy/Kobra_Plus#11]
I followed the steps as directed but was unable to get pronterface to connect to the printer. Now the printer won't communicate with the print nozzle.

**Comment by wabbitguy** [comment]
@Rage496 Anycubic uses a CH340G USB to Serial chip on the Trgorilla boards. If you don't have that driver install on your computer there's no way Pronterface will connect. Once the driver is installed, make sure you have the correct USB selected in Pronterface and the baud rate of 115200.

Look for the drivers from reliable sources like this one: [CH340G driver](https://www.arduined.eu/ch340-windows-10-driver-download/)

When the printer doesn't "talk" to the hot end (or all the other stuff in that area) I've found it's the cable from the motherboard to the hot end. It's either not seated in the connectors on each end, or, more often that I'd like it has broken connections in it.

**Comment by Rage496** [comment]
Unless I'm installing the drivers wrong it still wont connect all it does is says its connecting but does nothing.

**Comment by wabbitguy** [comment]
@Rage496 there's a drop down menu in Pronterface that give a list of USB ports available that it can connect to. There's nothing in the list that actually indicates it's connecting to a CH340G, a printer, or anything meaningful that I ever found.

On my Mac I get a bunch of dev/cu or dev/tty items listed, having never used Windows, I saw people post that they see COM4, COM5 etc.

The only video I found for connection is this one:

[Pronterface and Serial Driver](https://www.youtube.com/watch?v=MSdgzpIzy9Q)

**Comment by Rage496** [comment]
Everything I have input should be currect says connecting then I've tried to send the code and it says printer is offline

**Comment by wabbitguy** [comment]
@Rage496 when you connect to the printer should send out the entire list of current settings it has. If it sits at "connecting" but nothing shows up then the printer isn't responding. For any number of reasons. Port, baud rate, USB cable, USB hub to name a few. Until the printer is online and sends out the "Online" info, you can't send anything to it.

Having said that, I do know of more than one person that has had trouble with the USB ports on the Kobra series. They've even replaced the motherboards two or more times and it worked for a very short period of time and then it didn't work again. We never could figure out why it would work and then die. All we could guess at was a poor quality control inspection for the motherboards.

With the motherboard on my Max I used to be able run a print job from a server, so long as the print job was less than approximately 3.5hrs. Any longer than that and the printer would lock up, the USB port would disappear from the server and I'd have to reboot the printer to get the USB port back online. For longer print jobs I ended up using the SDcard.

Sorry I don't have any answer for you.

**Comment by Rage496** [comment]
All good I was hoping I missed a step somewhere but I think it's the printer itself just died

**Comment by wabbitguy** [comment]
@Rage496 I know the feeling. I've had so many printers simply stop working for any number of reasons over the last decade I've lost count. So far the only printers I still have working are some scratch built deltas I made in 2015, Anycubic Chiron 2018, and a PRUSA MK3S from 2019. The deltas and PRUSA have over 9,000hrs of print time on them. I know they are going to pack it in at some point but til then, they just keep printing.

Heck I even bought a Bambu X1C that was a lemon from day one based on everyone's positive feelings about it. I finally got it working after 4 months of tech hassles and then sold it.

The whole thing reminds me of buying appliances...LOL
